gradle/libs.versions.toml

```diff
@@ -1,6 +1,6 @@
 [versions]
 kotlin = "2.0.0"
-kotlinx-metadata = "0.7.0"
+kotlinx-metadata = "2.0.0"
```

Comment:
small thing, but should this not be changed to kotlin-metadata, same for kotlinx-metadataJvm